gazebo_ros/include/gazebo_ros/executor.hpp

```diff
@@ -34,18 +36,65 @@ class Executor : public rclcpp::executors::MultiThreadedExecutor
   /// Stops the internal executor and joins with the internal thread
   virtual ~Executor();
 
-private:
-  /// Thread where the executor spins until destruction
-  std::thread spin_thread_;
+  /// Add a node to the executor.
+  /**
+   * This is a thread-safe wrapper around rclcpp::Executor::add_node.
+   *
+   * \see rclcpp::Executor::add_node
+   */
+  virtual void
```

Comment:
nit: this should be `override` and not `virtual`, same to the other methods here

Reply:
see ad69f15